src/drem/transform/cso_sa.py

```diff
@@ -0,0 +1,70 @@
+import pandas as pd
+import geopandas as gpd
+from drem.filepaths import RAW_DIR
+from geodir.py import df_output
+
+# Spatially linking BER dataset to Geodir on SA level
+ber = pd.read_parquet(RAW_DIR / "BER_Closed.parquet")
+ber_dub = ber[ber["CountyName2"].str.contains("DUBLIN")]
+ber_dub = ber_dub.drop_duplicates()
+ber_extracted = ber_dub[
+    [
+        "cso_small_area",
+        "CountyName2",
+        "Year of construction",
+        "Year of construction range",
+        "Dwelling type description",
+        "Energy Rating",
+    ]
+]
+ber_renamed = ber_extracted.rename(columns={"CountyName2": "Dublin Postcode"})
+
+census = gpd.read_file(RAW_DIR / "Census2011_Small_Areas_generalised20m.shp")
+census_extracted = census[["SMALL_AREA", "geometry"]]
+census_renamed = census_extracted.rename(columns={"SMALL_AREA": "cso_small_area"})
+
+ber_sa = pd.merge(ber_renamed, census_renamed, on="cso_small_area", how="inner")
+ber_sa.sort_values("cso_small_area")
```

Comment:
```suggestion
ber_sa = ber_sa.sort_values("cso_small_area")
```
Otherwise won't do anything (I don't think) as haven't assigned it explicitely with inplace or equals operator